README.md

```diff
@@ -86,6 +86,9 @@ This is a work in progress, so if you know of Twitter tools that are not mention
 + [Personalized Recommendation of Twitter Lists Using Content and Network Information](https://www.aaai.org/ocs/index.php/ICWSM/ICWSM14/paper/download/8120/8142)
 + [Detecting Automation of Twitter Accounts: Are You a Human, Bot, or Cyborg?](https://www.ijirset.com/upload/2017/august/286_Implemnattaion_IEEE.pdf)
 + [Fame for sale: Efficient detection of fake Twitter followers](http://daneshyari.com/article/preview/553058.pdf)
++ [Sentiment Analysis of Twitter Data: A Survey of Techniques](https://arxiv.org/abs/1601.06971)
```

Comment:
Please remove this one. There is no point in adding multiple survey papers on the same topic